src/Particle.js

```diff
@@ -111,6 +111,14 @@ class Particle {
 	}
 
 	/**
+	 * Revoke the current session access token
+	 * @return {Promise}
+	 */
+	deleteCurrentAccessToken({ context }) {
+		return this.delete('/v1/access_tokens/current', {}, {}, context);
```

Comment:
You still need to pass the token as an argument. particle-api-js doesn't remember the token from one request to the other.

Reply:
But the API route does not accept a token for this endpoint, what gives? just drop it?